app/controllers/homebuyings_controller.rb

```diff
@@ -42,40 +42,49 @@ def create
   end
 
   def show
-    @homebuying = Homebuying.find(params[:id])
+    @homebuying = Homebuying.find(params[:id]) if current_client
```

Comment:
This is probably for the developer who works on the story: "As a `user` I can see a client's foreclosure app" ...but right now only a `user`can visit the foreclosure show page?